src/main/java/model/member/ExtendedBattleMember.java

```diff
@@ -846,10 +842,10 @@ private boolean usesJewellery() {
     }
 
     private boolean isNotAbleToUseSpells() {
-        return !characterisation.isAbleToUseSpells() ||
-                !race.isAbleToUseSpells() ||
-                !profession.isAbleToUseSpells() ||
-                !fightingStyle.isAbleToUseSpells() ||
+        return !characterisation.isAbleToUseSpells() &&
+                !race.isAbleToUseSpells() &&
+                !profession.isAbleToUseSpells() &&
+                !fightingStyle.isAbleToUseSpells() &&
```

Comment:
why did this change? With this, a non-magic race could use magic, if he spawns as a mage or is Enchanter or something.
Will constraints such as race be considered when generating fightingStyle or profession?

Reply:
It was just wrong in the first place.
If one of his trades (race, profession, ...) defines that he can use spells, he is able to use them.